rpmrc.in

```diff
@@ -80,6 +80,10 @@ optflags: armv6hl -O2 -g -march=armv6 -mfloat-abi=hard -mfpu=vfp
 optflags: armv7l -O2 -g -march=armv7
 optflags: armv7hl -O2 -g -march=armv7-a -mfloat-abi=hard -mfpu=vfpv3-d16
 optflags: armv7hnl -O2 -g -march=armv7-a -mfloat-abi=hard -mfpu=neon
+optflags: armv8l -O2 -g -march=armv8-a
```

Comment:
Do you implicitly really means -mfloat-abi=soft ? I anyone still care about this today ?

Reply:
aarch32 is the new marketing name for what we used to call the 'arm instruction set'.  armv8 is the most current incarnation of the 32-bit ARM instruction set.

While the armv8 spec strongly suggests the inclusion of the VFP/Neon and other instructions.  It is not strictly required.   So yes, softfloat is still being used and is needed in some cases.

There are multiple ARM licenses who claim to have armv8 compatible CPUs that do not have floating point units.  (Note, all of the cases I know of these are 32-bit CPUs -- and all are geared toward embedded.)

Reply:
@kwizart yes, implicit still is -mfloat-abi=soft (both in clang and gcc) even though I haven't seen that used on any general purpose CPU in ages -- nobody wants to change what everyone has come to expect.